internal/digreflect/func.go

```diff
@@ -66,9 +66,15 @@ func (f *Func) Format(w fmt.State, c rune) {
 // function.
 func InspectFunc(function interface{}) *Func {
 	fptr := reflect.ValueOf(function).Pointer()
-	f := runtime.FuncForPC(fptr)
+	return InspectFuncPC(fptr)
+}
+
+// InspectFuncPC inspects and returns runtime information about the function
+// at the given program counter address.
+func InspectFuncPC(pc uintptr) *Func {
+	f := runtime.FuncForPC(pc)
```

Comment:
Can we add a check for `f` being `nil` here? Upon an invalid PC we'll panic in the next line.